src/components/user/UserCardComponent.tsx

```diff
@@ -75,6 +78,9 @@ function UserCardComponent() {
             Email: {user?.email}
           </Typography>
         </Box>
+        <Box>
+          <Button onClick={() => navigate(ROUTE.SETTINGS)}>Settings</Button>
```

Comment:
It would be better here to use Link (ParLink as styled version of course) component instead of button, because its actually link. And you won't need a navigate.